x-pack/plugins/ml/public/application/datavisualizer/file_based/components/import_settings/simple.js

```diff
@@ -63,6 +64,7 @@ export const SimpleSettings = ({
         checked={createIndexPattern === true}
         disabled={initialized === true}
         onChange={onCreateIndexPatternChange}
+        data-test-subj="mlFileDataVistCreateIndexPatternCheckbox"
```

Comment:
Typo!? Should just be `mlFileDataVisCreateIndexPatternCheckbox`

Reply:
Good spot! Fixed in a0d3fef